zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -760,8 +760,13 @@ private static boolean publishDatasets(Collection collection) throws IOException
 
     private static void sendToKafka(Collection collection) throws IOException {
         List<String> uris = collection.getReviewed().uris();
+        List<String> validUris = uris.stream().map (temp -> {
+            convertUriForEvent(temp);
+            return temp;
+        }).collect(Collectors.toList());
```

Comment:
This isn't going to do what you think it will. The lambda is calling the conversion method but then returning the original uri rather than the converted string. The original code I suggested is simpler and should work instead.
```suggestion
        List<String> uris = collection.getReviewed().uris()
                .stream().map((temp) -> convertUriForEvent(temp))
                .collect(Collectors.toList());
```